examples/environment-variables/environment-variables.go

```diff
@@ -11,21 +12,24 @@ import "fmt"
 
 func main() {
 
-    // To set a key/value pair, use `os.Setenv`. To get a
-    // value for a key, use `os.Getenv`. This will return
-    // an empty string if the key isn't present in the
-    // environment.
-    os.Setenv("FOO", "1")
-    fmt.Println("FOO:", os.Getenv("FOO"))
-    fmt.Println("BAR:", os.Getenv("BAR"))
+	// Per impostare una coppia chiave/valore, usa
```

Comment:
Secondo me qui è poco chiaro. Sarebbe meglio cambiare in
`Per impostare una variabile d'ambiente tramite una coppia chiave/valore`

o qualcosa di simile